src/tests/unit/services/CicService.test.ts

```diff
@@ -60,13 +60,31 @@ describe("Cic Service", () => {
 
 	it("should throw 500 if request fails during save CIC data", async () => {
```

Comment:
positive case needed as well when no error is thrown and single names provided

Reply:
Positive case added